Calculator/Formula.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  Formula.swift
+//  Calculator
+//
+//  Created by Yetti on 2023/06/09.
+//
+
+struct Formula {
+    var operands: CalculatorItemQueue<Double> = CalculatorItemQueue()
+    var operators: CalculatorItemQueue<Operator> = CalculatorItemQueue()
+    
+    mutating func result() throws -> Double {
+        var result: Double = 0.0
+        guard var lhs: Double = operands.dequeue() else { throw CalculatorError.inValidCalculate }
+        
+        while operands.isEmpty == false {
+            let dequeueOperator = operators.dequeue()
+            
+            guard let rhs = operands.dequeue(),
+                  let calculatedValue = dequeueOperator?.calculate(lhs: lhs, rhs: rhs) else { throw CalculatorError.inValidCalculate }
+            
+            lhs = calculatedValue
+            result = calculatedValue
+        }
+        return result
+    }
```

Comment:
`isEmpty`를 통해 큐가 빌 때까지 작업을 반복하고 있군요! 한가지 팁을 드리자면 `while`문도 조건문이기 때문에 옵셔널 바인딩이 가능합니다! 다음과 같이 쓸 수 있죠! 
```suggestion
    mutating func result() throws -> Double {
        guard var result = operands.dequeue() else { throw CalculatorError.inValidCalculate }
        
        while let rhs = operands.dequeue(),
              let dequeuedOperator = operators.dequeue() {
            result = dequeuedOperator.calculate(lhs: result, rhs: rhs)
        }
        
        return result
    }
```
어떤가요? 불필요한 변수 선언도 줄일 수 있고, `isEmpty`를 굳이 사용하지 않아도 되며, guard문을 통한 바인딩이 생략되죠? 😁